### PR TITLE
Added requestData to get results from BQ tables

### PR DESCRIPTION
**Pull request by mdemauroy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,11 +144,40 @@ To automatically register your metadata into the UnitOfWork, this bundle provide
 At this point your metadata are declared into the UnitOfWork, thanks to a CompilerPass.
 You are ready to upload data.
 
-### Upload data to google bigquery
-To upload data, you need to use the service `CCMBenchmark\BigQueryBundle\BigQuery\UnitOfWork`.
-This service offers a simple API. Call `addData` to store a new Entity to upload.
+### Working with UnitOfWorl service
```
Comment (xavierleune) [inline README.md]: Typo: `UnitOfWork`

File: README.md [hunk 2]
```diff
@@ -144,11 +144,40 @@ To automatically register your metadata into the UnitOfWork, this bundle provide
...
+
+#### Upload data to google bigquery
+Call `addData` to store a new Entity to upload.
 When all you're entities are in the UnitOfWork, call `flush` to upload it.
 
+#### Request data from google bigquery
+Call `requestData` to make a request to the specified `projectId` it will return a `\Google\Service\Bigquery\GetQueryResultsResponse`
+```php
+<?php
+class myDataSource implements DataSourceInterface
```
Comment (xavierleune) [inline README.md]: `DataSourceInterface` does not belongs to this project, it shoud not be referenced in this example

File: src/BigQueryBundle/BigQuery/UnitOfWork.php [hunk 3]
```diff
@@ -7,6 +7,7 @@
 use CCMBenchmark\BigQueryBundle\BigQuery\Entity\RowInterface;
 use CCMBenchmark\BigQueryBundle\CloudStorage\FileSystem;
 use CCMBenchmark\BigQueryBundle\CloudStorage\FileSystemInterface;
+use Google_Service_Bigquery_QueryRequest;
```
Comment (xavierleune) [inline src/BigQueryBundle/BigQuery/UnitOfWork.php]: The clients now offers a proper namespace, can you rely on `Google\Service\BigQuery\QueryRequest` (double check this name) instead ?

**Review by xavierleune — CHANGES_REQUESTED** [review]
Great ! A few typo to fix but this looks good to me

File: README.md (hunk 1 above)
Comment (mdemauroy) [reply README.md]: Already fixed ^^

**Review by xavierleune — APPROVED** [review]

**Pull request merged**